python/lsst/pipe/analysis/utils.py

```diff
@@ -2015,3 +2153,52 @@ def calcQuartileClippedStats(dataArray, nSigmaToClip=3.0):
         clipValue=clipValue,
         goodArray=good,
     )
+
+
+def corr2(ra, dec, g1a, g2a, g1b=None, g2b=None, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
+    """ Function to compute auto-/cross-correlations between atmost two shear fields.
```

Comment:
Sorry to keep nitpicking over this, but I really did like your idea of saying "shear-like" fields here (as you do below); even moreso now that it's in utils.py